### PR TITLE
Spotlights June 2017 

### PR DESCRIPTION
**Pull request by Shadowthegod**

### CODE REVIEW

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Beatmap Spotlights: June 2017"
+permalink: 
+date: 2017-07-01 14:50:00 +0000
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Change date to 2017-07-22 13:30:00 +0000

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Beatmap Spotlights: June 2017"
+permalink:
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: put `permalink: http://osunews.tumblr.com/post/163285435403/beatmap-spotlights-june-2017`

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Beatmap Spotlights: June 2017"
+permalink: 
+date: 2017-07-01 14:50:00 +0000
+tumblr_url:
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Put `tumblr_url: http://osunews.tumblr.com/post/163285435403/beatmap-spotlights-june-2017`

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Beatmap Spotlights: June 2017"
+permalink: 
+date: 2017-07-01 14:50:00 +0000
+tumblr_url: 
+---
+
+
+Demetori, Infected Mushroom, Seiryu and KRUX reign supreme in the [Beatmap Spotlights](https://osu.ppy.sh/p/chart?ch=MONTH1705) of June 2017, and more to boot! Come check out what's in store this month, and maybe even win an osu!supporter tag for your trouble!
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Should be `MONTH1706`. Also maybe come up with a different intro sentence and don't recycle the same from every month.

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+Absolute mastery of a style at its finest. **[GoldenWolf](https://osu.ppy.sh/u/1612624)**'s latest marathon takes us on a journey through this epic piece by Demetori. The structure of the map is simple, yet refined. The gameplay experience is difficult, yet accompanies the song perfectly. And not to mention the classic meticulous GoldenWolf hitsounding! A well-deserved top placing on this month's charts.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/mahou-shoujo-chino.jpg)](https://osu.ppy.sh/s/458983)
+
+After No Poi, **[Asahina Momoko](https://osu.ppy.sh/u/3650145)** returns again in 2017 to bring us more Chino. And good mapping. Asahina Momoko never sticks to the 'mapping-meta', and as such, all his maps are always different and interesting, With his particular style, he's managed to capture this song perfectly with strong innovative spacing usage to create different intensities. His rhythm usage is top notch, being consistent enough to maintain playability, while having ample variation so it does not bore out the player with its length. You won't regret giving this one a good play through!
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: > And good mapping.

Should be " - and good mapping." Remove the dot after Chino for that, too.

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/hyadain.jpg)](https://osu.ppy.sh/s/554297)
+
+This month, **[Okorin](https://osu.ppy.sh/u/1623405)** brings us a more modern re-map of a song that has already been ranked multiple times before. The higher difficulties captivate with fun jumps and well-placed symmetry usage. The lower difficulties aren't of less quality though, since they manage to mirror the interesting rhythm and gameplay elements from the higher difficulties while still being easy enough. This beautiful spread is worth checking out for players of every skill level!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/innocence.jpg)](https://osu.ppy.sh/s/557231)
+
+Calm, yet very intense - this describes both song and mapset. appleeaterx has fulfilled a great job in emphasizing the right spots, to give the player the best feeling possible to fully enjoy this song. Definitely give it a try!
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Link to appleeaterx' profile missing

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+This seven minute long map features all kinds of things, from typical 1/4-snapped sections (the common snapping in Oni difficulties), to more tricky combinations with 1/6 and 1/8 snaps, mixed together with a high amount of slider velocity changes to highlight the varying power of each of the song's sections. He also does not fear to use a pattern that used to be common, but has been declined in the past years for other patterns: Long 1/6. All around a worthy number one slot for this month's charts.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/teriquma.jpg)](https://osu.ppy.sh/s/615774)
+
+**[Charlotte](https://osu.ppy.sh/u/3686901)** sure has been present in the taiko spotlights these last months, and rightfully so! Once again providing the ranked section with a great mapset, featuring as always a flawless structure and a perfect 1/6 snapping usage in the Inner Oni, it grealy deserves the attention of taiko players this month!
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: > taiko

osu!taiko

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/blinded-in-chains.jpg)](https://osu.ppy.sh/s/536955)
+
+According mapping experience, **[gaston_2199](https://osu.ppy.sh/u/5938161)** is the least experienced taiko mapper among nominated mappers, but it doesn't impact the quality of this map. The map contains quite solid structure with common pattern. However, it doesn't make the map become boring. He uses different combination of patterns but same structure to distinguish the difference of melody precisely. Beside that, the highlight of this map is streams. He also adopts similar idea, and hitsounds fit guitar, drum and even vocal pretty well. In conclusion, the map is well structured and could be a good model to learn how to map taiko map on this kind of song.
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Again osu!taiko, not taiko

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/we-are-number-one.jpg)](https://osu.ppy.sh/s/547301)
+
+"Now listen closely...here's a little lesson on this mapping spree. This is going down in history! If you wanna be a beatmap number one, you have to chase a nominator on the run!"
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Yeh, not a fan of this one.

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+In his flagship Number Oni difficulty, Midnaait shows off the most diverse range of technical snappings. Particularly impressive to note was his stylistic placement of 1/8th doublets throughout - this complemented well with the quirky nature of the song, and gave it an extra "kick" to make it all the more enjoyable.
+
+## osu!catch
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/ultramarine.jpg)](https://osu.ppy.sh/s/275991)
+
+With enjoyable mapping and unique rhythms, **[Zirox](https://osu.ppy.sh/u/200768)** and fellow collaborators **[Ascendance](https://osu.ppy.sh/u/2931883)** and **[Spectator](https://osu.ppy.sh/u/702598)** have created a map that challenges the player's reading skills yet is consistent. Spec's Abyss provides a pleasant reading challenge with CS 5 as well as some custom hitsounds that add to the atmosphere of the map. Rain catches attention with it's lower AR of 8.7 and dash patterns that are usually uncommon in the current meta. While Cup, Salad and Platter are more of the standard meta, they are still enjoyable to play.
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: > CS

Circle Size

> AR

Approach Rate

Avoid abbreviations.

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/exit-this-earth-atomosphere.jpg)](https://osu.ppy.sh/s/575053)
+
+It's always a good surprise to find an amazing mapset as a first ranked map, and this is the case with **[Protastic101](https://osu.ppy.sh/u/6712747)**'s set! This set shows a lot of diversity: Burst, Jumpstreams, long notes... But the most amazing point of the set is the usage of Slider Velocity, empathizing a lot of slower parts and creating a really powerful feeling on the harder ones. Overall, it creates a really challenging and technical map set, sprinkled with a lot rare patterns seldom seen in the ranked section.
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: A triple dot is not really correct. Rather say `Burst, Jumpstreams, long notes - but the most amazing`

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/rock-it.jpg)](https://osu.ppy.sh/s/569903)
+
+This map is certainly a surprise for me. Kicking off with a slowjam that slowly speeds up to your usual scroll building up the momentum with a no stop low density handstream into a technical jump-gallops with really flowy dense bursts. This map is overall a pretty nice piece hidden there in the ranked pool and I really wanted to give it the chance to be featured here. Overall just a really fun accuracy map with great technical elements and eventually some mini long notes and slider velocity changes. Fear not for those who are not capable to play this since the next difficulty from Eedow is as good an as fun to play accuracy wise but just in a lower level. Well deserved spot overall for a solid mapset.
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: > Slowjam

I think this is a word that nobody understands

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+--------
+
+And that just about covers the Spotlights for this month. We hope you enjoy the new template graphics for each chart, and we hope you look forward to what we have in store next month as well!
+
+—OnosakiHito
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Not Kurokami? Or did Ono compile this post? ;)

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/rock-it.jpg)](https://osu.ppy.sh/s/569903)
+
+This map is certainly a surprise for me. Kicking off with a slowjam that slowly speeds up to your usual scroll building up the momentum with a no stop low density handstream into a technical jump-gallops with really flowy dense bursts. This map is overall a pretty nice piece hidden there in the ranked pool and I really wanted to give it the chance to be featured here. Overall just a really fun accuracy map with great technical elements and eventually some mini long notes and slider velocity changes. Fear not for those who are not capable to play this since the next difficulty from Eedow is as good an as fun to play accuracy wise but just in a lower level. Well deserved spot overall for a solid mapset.
+
+--------
+
+And that just about covers the Spotlights for this month. We hope you enjoy the new template graphics for each chart, and we hope you look forward to what we have in store next month as well!
```
Comment (Loctav) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: I think we can now remove that or replace that with some better paragraph

File: news/2017-07-22-beatmap-spotlight-june-2017.md (hunk 9 above)
Comment (mangomizer) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Nooooooooooo

File: news/2017-07-22-beatmap-spotlight-june-2017.md (hunk 13 above)
Comment (Shadowthegod) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: It was me but we agreed that monthly will go under his name and mine will be the themed ones (soon™).

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/we-are-number-one.jpg)](https://osu.ppy.sh/s/547301)
+
+Following the immense popularity of the song "We Are Number One", this Dunderpartrullen remix offers an even catchier rhythm in addition to the already memorable tune. **[Midnaait](https://osu.ppy.sh/u/1506011)**'s mapset capitalizes on this extremely well, offering a wide selection of fun difficulties to play, with contributions from guest mappers Lno and Kantan.
+
+In his flagship Number Oni difficulty, Midnaait shows off the most diverse range of technical snappings. Particularly impressive to note was his stylistic placement of 1/8th doublets throughout - this complemented well with the quirky nature of the song, and gave it an extra "kick" to make it all the more enjoyable.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: this one's gooooooood

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+
+## osu!catch
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/ultramarine.jpg)](https://osu.ppy.sh/s/275991)
+
+With enjoyable mapping and unique rhythms, **[Zirox](https://osu.ppy.sh/u/200768)** and fellow collaborators **[Ascendance](https://osu.ppy.sh/u/2931883)** and **[Spectator](https://osu.ppy.sh/u/702598)** have created a map that challenges the player's reading skills yet is consistent. Spec's Abyss provides a pleasant reading challenge with Circle Size 5 as well as some custom hitsounds that add to the atmosphere of the map. Rain catches attention with it's lower Approach Rate of 8.7 and dash patterns that are usually uncommon in the current meta. While Cup, Salad and Platter are more of the standard meta, they are still enjoyable to play.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: Spec's -> Spectator's
Circle Size -> circle size
Approach Rate -> approach rate

more of the standard meta -> more standard offerings gameplay wise

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/mahou-shoujo-chino_osucatch.jpg)](https://osu.ppy.sh/s/458983)
+
+Making an appearance alongside **[Asahina Momoko](https://osu.ppy.sh/u/3650145)**'s spotlight worthy osu! difficulty, veteran mapper **[Minako Yukina](https://osu.ppy.sh/u/531253)** (previously known as CLSW) showcases his experience of catch mapping with a map that does a wonderful job of expressing the intricate rhythms of this cute and upbeat song. While it's not quite at the level of difficulty many players have to come know his maps for its still sure to provide a challenge for all but the best catch players.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: yowza this one's good

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/syncretism.jpg)](https://osu.ppy.sh/s/514144)
+
+It takes skill to make an interesting map out of a simple song, but **[RoseusJaeger](https://osu.ppy.sh/u/6709840)** has achieved that nicely here. Each difficulty features a variety of instrumental and vocal rhythm choices, layered together in a clean, fun and intuitive way, resulting in a beatmap set that is relaxing and fun to play for players of all skill levels. The creative use of 1/6 patterns adds some variety and character, helping this set to stand out among the many available choices in what has undoubtedly been a very productive month for osu!catch mapping.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: okay no seriously sign me the fuck up for more of this

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/moonlight-tango.jpg)](https://osu.ppy.sh/s/536001)
+
+Mapping lower BPM certainly brings its own challenges in osu!catch but **[JBHyperion](https://osu.ppy.sh/u/4879508)** has made it look near effortless here. Making excellent use of absolutely everything at his disposal, dashes, hyperdashes, anti-flow and some fairly unique stream patterns, it all comes together to represent the music fantastically whilst still being a real delight to play across all difficulties. Once again he delivers a level of quality every catch mapset should aspire to achieve.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: STOP YOU'RE MAKING THE OTHERS LOOK BAD

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/illusion-of-inflict.jpg)](https://osu.ppy.sh/s/529574)
+
+This time, **[Reba](https://osu.ppy.sh/u/3087654)** is bringing us an almost perfectly structured beatmap in every aspect as well as seven guest difficulties. Although the BPM is not really high, you can find challenging patterns such as jacks, long notes, chordstreams and more, specially in **[Critical_Star](https://osu.ppy.sh/u/3793196)**'s. Talking about the hardest map in the set, it has everything mentioned before but way more dense than all the other diffculties as well as a few noticeable Slider Velocity changes at the last section of the map. Highly recommended to play since it has difficulties for every type of player, going from 1 star to almost 6 and a half.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: new paragraph at Talking about

Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: recommended line is a bit awkwardly phrased but passable

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 21]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/exit-this-earth-atomosphere.jpg)](https://osu.ppy.sh/s/575053)
+
+It's always a good surprise to find an amazing mapset as a first ranked map, and this is the case with **[Protastic101](https://osu.ppy.sh/u/6712747)**'s set! This set shows a lot of diversity: Burst, Jumpstreams, long notes - but the most amazing point of the set is the usage of Slider Velocity, empathizing a lot of slower parts and creating a really powerful feeling on the harder ones. Overall, it creates a really challenging and technical map set, sprinkled with a lot rare patterns seldom seen in the ranked section.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: swap colon after diversity for a comma (or use 'with' instead), don't capitalize mapping technique names (they're not proper nouns)

empathizing -> emphasizing

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 22]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/rock-it.jpg)](https://osu.ppy.sh/s/569903)
+
+This map is certainly a surprise for me. Kicking off with a low scroll speed section that slowly speeds up to your usual scroll building up the momentum with a no stop low density handstream into a technical jump-gallops with really flowy dense bursts. This map is overall a pretty nice piece hidden there in the ranked pool and I really wanted to give it the chance to be featured here. Overall just a really fun accuracy map with great technical elements and eventually some mini long notes and slider velocity changes. Fear not for those who are not capable to play this since the next difficulty from Eedow is as good an as fun to play accuracy wise but just in a lower level. Well deserved spot overall for a solid mapset.
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: needs to reference the mapper in bold to stay consistent with the other reccomendations

remove "this map is an overall pretty nice piece" sentence

remove "overall just a really fun accuracy map" sentence

Probably need to squish the content from those sentences into the ending sentence instead

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 23]
```diff
@@ -0,0 +1,106 @@
...
+
+--------
+
+And that is all we had in the Spotlights this month. We hope you enjoy the mapsets we offered this time and we hope you look forward to what we have in store next month as well!
```
Comment (Ephemeralis) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: And that's all we've got in the Spotlights for this month.

File: news/2017-07-22-beatmap-spotlight-june-2017.md [hunk 24]
```diff
@@ -0,0 +1,106 @@
...
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/hyadain.jpg)](https://osu.ppy.sh/s/554297)
+
+This month, **[Okorin](https://osu.ppy.sh/u/1623405)** brings us a more modern re-map of a song that has already been ranked multiple times before. The higher difficulties captivate with fun jumps and well-placed symmetry usage. The lower difficulties aren't of less quality though, since they manage to mirror the interesting rhythm and gameplay elements from the higher difficulties while still being easy enough. This beautiful spread is worth checking out for players of every skill level!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/innocence.jpg)](https://osu.ppy.sh/s/557231)
+
+Calm, yet very intense - this describes both song and mapset. **[appleeaterx](https://osu.ppy.sh/u/2407160)** has fulfilled a great job in emphasizing the right spots, to give the player the best feeling possible to fully enjoy this song. Definitely give it a try!
```
Comment (Shadowthegod) [inline news/2017-07-22-beatmap-spotlight-june-2017.md]: I changed the second sentence a little to might make it clearer but if you want further changes you have to speak with Irreversible as I won't re-write the entire comment, and he doesn't seem to understand what you expect from him here.

**Pull request merged**